src/featurestorebundle/delta/feature/NullHandler.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from pyspark.sql import DataFrame
+from pyspark.sql import functions as f
+from featurestorebundle.feature.FeatureList import FeatureList
+
+
+class NullHandler:
+    def fill_nulls(self, df: DataFrame, feature_list: FeatureList) -> DataFrame:
+        self.__check_default_values_valid(feature_list)
+
+        fill_dict = {
+            feature.name: feature.template.default_value for feature in feature_list.get_all() if feature.template.default_value is not None
+        }
+
+        return df.fillna(fill_dict)
+
+    def to_storage_format(self, df: DataFrame, feature_list: FeatureList) -> DataFrame:
+        for feature in feature_list.get_all():
+            if feature.template.default_value is None:
+                df = df.withColumn(feature.name, f.create_map(f.lit(0), f.col(feature.name)).cast(feature.storage_dtype))
+
+        return df
+
+    def from_storage_format(self, df: DataFrame, feature_list: FeatureList) -> DataFrame:
+        for feature in feature_list.get_all():
+            if feature.template.default_value is None:
+                df = df.withColumn(feature.name, f.col(feature.name).getItem(0))
+
+        return df
+
+    def __check_default_values_valid(self, feature_list: FeatureList):
+        for feature in feature_list.get_all():
+            if feature.template.default_value in [np.nan, np.inf]:
```

Comment:
there is also negative infinity np.ninf, but I would rather use built-in float("nan"), float("inf") and float("-inf"), they are equivalent and we don't have to depend on numpy

Reply:
are they really equivalent? Maybe after conversion to dataframe
![image](https://user-images.githubusercontent.com/22962839/161025551-4f534d5d-c814-42c9-b4dc-bba106492707.png)

Reply:
looks like we have to do this
```
if feature.template.default_value in [np.nan, np.inf, -np.inf, float("nan"), float("inf"), float("-inf")]:
```

Reply:
or `math.isnan(default_value) or math.isinf(default_value)`